reports/sklearn_estimators_sample_weight_audit_report.ipynb

```diff
@@ -1588,6 +1587,7 @@
     "            est_name,\n",
     "            test_name=\"kstest\",\n",
     "            n_stochastic_fits=N_STOCHASTIC_FITS,\n",
+    "            store_estimators=True,\n",
```

Comment:
I think we shouldn't store the estimators when running the check on all estimators in scikit-learn.

But we should definitely enable this option in the other notebook where we investigate a single estimator at a time.

Reply:
I think you can revert the changes made to this notebook and instead do the change in the other.